src/wallet/spend.cpp

```diff
@@ -346,6 +351,15 @@ CoinsResult AvailableCoins(const CWallet& wallet,
         }
     }
 
+    if (feerate.has_value()) {
+        std::map<COutPoint, CAmount> map_of_bump_fees = wallet.chain().CalculateIndividualBumpFees(outpoints, feerate.value());
+
+        for (auto& output : result.All()) {
+            output.ApplyBumpFee(map_of_bump_fees.at(output.outpoint));
```

Comment:
In f2daebf8d1ac3829faa6443425cb97a763e9f780 "Bump unconfirmed parent txs to target feerate"

This doesn't actually apply the bumpfees to all of the outputs in `result. `All()` will return a copy of all of the outputs, not references to them, so this is just applying the bumpfees to a temporary.

Because of this, the tests added in this commit fail. They only do not fail because of the adjustment to the fee that is done in the following commit.